### DISCUSSION TITLE
双拼中，能不能自造一个拼音？

### DISCUSSION THREAD

**Discussion by kwindva** [rime/home#1720] [Q&A] [answered]
想把「䊆」这个字加上「kiu」的音，然后能用拼音「kiu　tan」打出「䊆弹」（代替「Q弹」）。
在cn_dicts下面的others.dict.yaml里加上：
䊆  kiu
䊆弹  kiu tan

重新部署，不起作用。。打「k」和「j」（我的双拼方案里，iu对应j），依然是当成了两个不能拼在一起的韵母。

**Comment by lotem** [comment, marked as answer]
雙拼方案的拼寫運算規則轉換不了這個音節。
超出音系了。官話聲母 g k h 不拼細音。

一定要拼，聲母應當會顎化，折合成 qiu。
拼 qiu 就不用改規則了。

**Reply by lotem** [reply]
另外你說的`cn_dicts` 我不瞭解。應該不是輸入法自帶的方案。最好諮詢方案作者。

**Reply by kwindva** [reply]
多谢，我在custom_phrase.txt里面，直接写上：
䊆	kj
䊆	kjt
䊆弹	kjtd
反正这个音也就只有这个字，也就只有这个词。。
